src/main/java/atdd/path/web/UserController.java

```diff
@@ -16,17 +20,18 @@
 @RequestMapping("/users")
 public class UserController {
     private UserRepository userRepository;
+    private JwtTokenProvider jwtTokenProvider;
 
-    public UserController(UserRepository userRepository) {
+    public UserController(UserRepository userRepository, JwtTokenProvider jwtTokenProvider) {
         this.userRepository = userRepository;
+        this.jwtTokenProvider = jwtTokenProvider;
     }
 
     @PostMapping
     public ResponseEntity createUser(@RequestBody CreateUserRequestView view) {
```

Comment:
Java 제네릭을 사용하면 어떨까요? 아래의 글이 도움 되면 좋겠어요.
https://madplay.github.io/post/dont-use-raw-types

```suggestion
    public ResponseEntity<UserResponseView> createUser(@RequestBody CreateUserRequestView view) {
```